validator/sawtooth_validator/state/settings_view.py

```diff
@@ -47,8 +47,13 @@ def __init__(self, state_view):
         """
         self._state_view = state_view
 
-    @lru_cache(maxsize=128)
-    def get_setting(self, key, default_value=None, value_type=str):
+        # The public method for get_settings should have its results  memoized
```

Comment:
extra space between results and memoized

Reply:
so you're going to call out an extra space but not a double the?